config/config.go

```diff
@@ -55,6 +55,16 @@ const (
 	// SettingDebugLogDefault is the default value for the debug log enabling
 	SettingDebugLogDefault = false
 
+	// SettingInventoryURI is the config key for the inventory uri
+	SettingInventoryURI = "inventory_uri"
```

Comment:
some services use UIR some URL, this one seems consistent on the URL suffix. just truing to turn your attention on this one, feel free to disregard.

Reply:
done